btrack/napari/constants.py

```diff
@@ -26,5 +16,4 @@
     "dist_thresh",
     "time_thresh",
     "apop_thresh",
-    "relax",
```

Comment:
I'm going to merge this now, but just to check. @p-j-smith `relax` got removed because it should have never been in this list, right?